src/app/api/routes/alerts.py

```diff
@@ -59,3 +60,13 @@ async def link_media(payload: AlertMediaId,
         )
     existing_alert["media_id"] = payload.media_id
     return await crud.update_entry(alerts, AlertIn(**existing_alert), alert_id)
+
+
+@router.get("/ongoing", response_model=List[AlertOut])
+async def fetch_ongoing_alerts():
+    return await crud.fetch_all(alerts, {"type": AlertType.start})
```

Comment:
We may have multiple alerts launched for the same event_id?
From what you are saying the SQL query should be something like:

```SQL
SELECT * from 
    alerts a
where 
     type = AlertType.start
     and event_id not in (select event_id from alerts where type=Alert.end)

```

Reply:
Looks perfect to me!

Reply:
I added a new commit to handle that logic. There is probably a way to do it in a clenaer way using Sqlaclhemy models instead of calling the table directly. I think we should do it if we want to handle complex queries in a cleany way. 

However, for now it is working. I will move the created crud function after the refacto.